### ISSUE TITLE
UI and logic for projection mapping over a CS table 

### ISSUE THREAD

**Issue by RELNO** [CityScope/CSL_Hamburg_Grasbrook_Angular_Prototype#15]
This functionality should allow flat/3d projection and keystoning of the applet over a physical table. 

Constrains:
- mapbox extrusions have fixed perspective, not suited for 2d flat projection, therefore they should be flattened on projection mode 
- div size should fit table size [not screen ratio] 
- key-stone logic should include saving points between sessions

**Comment by aberke** [comment]
For the subtask of "extrusions are flattened when in 'projection mode'", how should we specify that we are in 'projection mode'?  A button?
Another way to do this is to have a URL parameter in the endpoint, like `http://tableURL.com?mode=projection-mode`

**Comment by RELNO** [comment]
@aberke yes -- I guess it will part of a UI dedicate to projection mapping on a table, so a master button will get the user into this mode and as a result, 3d =>2d.
